apps/sample-app/src/components/PeopleAvatars.tsx

```diff
@@ -59,48 +69,55 @@ export const PeopleAvatarList: React.FC<AvatarProps> = (props: AvatarProps) => {
   const AvatarExample: React.FunctionComponent = () => {
     const capabilities = getSupportedCapabilities();
     return (
-      <div>
-        {AvatarItemList.map(a => (
-          <Tooltip
-            content={
-              <>
-                <Text weight="semibold" as="span">
-                  {a.name}
-                </Text>
-                <MenuList>
-                  {capabilities.map(capability => (
-                    <div key={capability}>
-                      {capability === 'Call' && (
-                        <Menu>
-                          <MenuTrigger>
-                            <MenuItem>Call</MenuItem>
-                          </MenuTrigger>
-                          <MenuPopover>
-                            <MenuList>
-                              <MenuItem onClick={() => handleAudioCall(a)}>Audio {capability}</MenuItem>
-                              <MenuItem onClick={() => handleVideoCall(a)}> Video {capability}</MenuItem>
-                            </MenuList>
-                          </MenuPopover>
-                        </Menu>
-                      )}
-                      {capability === 'Message' && <MenuItem onClick={() => handleMessage(a)}> {capability}</MenuItem>}
-                      {capability === 'Mail' && <MenuItem onClick={() => handleMail(a)}>{capability}</MenuItem>}
-                    </div>
-                  ))}
-                </MenuList>
-              </>
-            }
-            key={a.id}
-            relationship={'label'}
-          >
-            <Text as="span" key={a.id}>
-              <button key={a.id}>
-                <Avatar key={a.id} {...a} color="colorful" size={56} />
-              </button>
-            </Text>
-          </Tooltip>
-        ))}
-      </div>
+      <>
+        <Title3 block className="paddingClass">
+          People to Meet Today
+        </Title3>
+        <div>
+          {AvatarItemList.map(a => (
+            <Tooltip
+              content={
+                <>
+                  <Text weight="semibold" as="span">
+                    {a.name}
+                  </Text>
+                  <MenuList>
+                    {capabilities.map(capability => (
```

Comment:
I think it's OK to leave it as is, but I'm curious why you think this is better than having the `isSupported` check here, something like:

```
{mail.isSupported() && <MenuItem onClick={() => handleMail(a)}>Mail</MenuItem>
```

Reply:
I agree that this is more readable and simpler to understand. I'll try incorporating it this way and see if the UI remains the same (which seems like it should!) and push it in a future PR?

Reply:
sounds good!